### PR TITLE
Small clarification to logging docs

### PR DESCRIPTION
**Pull request by martinzlocha**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
It was unclear that the ForwardingDispatcher doesn't send LogType.Log by default.

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/content/ecs/logging.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, `LogType.Log` isn't sent to SpatialOS Console. This can be changed by setting the minimumLogLevel when instantiating the `ForwardingDispatcher`
```
[the remark this answers is not in the document]
Comment (martinzlocha) [reply docs/content/ecs/logging.md]: Updated, please have a look.

File: docs/content/ecs/logging.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
```
[the remark this answers is not in the document]
Comment (martinzlocha) [reply docs/content/ecs/logging.md]: Do you know what would be a better word?

File: docs/content/ecs/logging.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
...
+    * Note: By default, `LogType.Log` is not sent to SpatialOS Console. This can be changed by instantiating the `ForwardingDispatcher` with different `minimumLogLevel` parameter
```
Comment (firtina-improbable) [inline docs/content/ecs/logging.md]: > with *a* different

missing period at end of line

File: docs/content/ecs/logging.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jessicafalk) [reply docs/content/ecs/logging.md]: actually had a discussion with Laura about this last week. I think the official terms used in docs are "Unity Console" and "SpatialOS console"

File: docs/content/ecs/logging.md (hunk 3 above)
Comment (jessicafalk) [reply docs/content/ecs/logging.md]: add an example?

File: docs/content/ecs/logging.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
-    * Note: By default, `LogType.Log` is not sent to SpatialOS Console. This can be changed by instantiating the `ForwardingDispatcher` with different `minimumLogLevel` parameter
+    * Note: By default, `LogType.Log` is not sent to SpatialOS Console. This can be changed by instantiating the `ForwardingDispatcher` with a different `minimumLogLevel` parameter.
```
Comment (jessicafalk) [inline docs/content/ecs/logging.md]: console should be lower case

File: docs/content/ecs/logging.md (hunk 3 above)
Comment (martinzlocha) [reply docs/content/ecs/logging.md]: I don't think that there should be an example here. Right below is a link to a section which talks about creating a dispatcher. Maybe I should rename it to "Modifying" instead of "Creating".

File: docs/content/ecs/logging.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, `LogType.Log` is not sent to SpatialOS Console. This can be changed by instantiating the `ForwardingDispatcher` with different `minimumLogLevel` parameter
 
 All workers use the `ForwardingDispatcher` by default. To replace it with the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
```
Comment (firtina-improbable) [inline docs/content/ecs/logging.md]: Driveby review, this sounds wrong:

> To replace it with the `LoggingDispatcher`

Comment (martinzlocha) [reply docs/content/ecs/logging.md]: Updated :)

File: docs/content/ecs/logging.md (hunk 4 above)
Comment (martinzlocha) [reply docs/content/ecs/logging.md]: Changed :)

File: docs/content/ecs/logging.md [hunk 6]
```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, `LogType.Log` is not sent to SpatialOS console. This can be changed by instantiating the `ForwardingDispatcher` with a different `minimumLogLevel` parameter.
```
Comment (jessicafalk) [inline docs/content/ecs/logging.md]: the first sentence should be rephrased a bit. maybe something like:

Note: by default, messages with log level `LogType.Log` are not send to SpatialOS.

File: docs/content/ecs/logging.md [hunk 7]
```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
...
 
-All workers use the `ForwardingDispatcher` by default. To replace it with the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
+All workers use the `ForwardingDispatcher` by default. If you want to use the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
```
Comment (jessicafalk) [inline docs/content/ecs/logging.md]: this is only true for the playground. need to make that more explicit

File: docs/content/ecs/logging.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JonasImprobable) [reply docs/content/ecs/logging.md]: The term "console" can be misleading for clouds deployments.

File: docs/content/ecs/logging.md [hunk 8]
```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, messages with log level `LogType.Log` are not send to SpatialOS. This can be changed by instantiating the `ForwardingDispatcher` with a different `minimumLogLevel` parameter.
```
Comment (firtina-improbable) [inline docs/content/ecs/logging.md]: messages with **the** log level ... are not **sent** to

File: docs/content/ecs/logging.md [hunk 9]
```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
...
 
-All workers use the `ForwardingDispatcher` by default. To replace it with the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
+All workers use the `ForwardingDispatcher` by default in the Playground. If you want to use the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
```
Comment (firtina-improbable) [inline docs/content/ecs/logging.md]: Nitpicking: I am wondering if we should mention the `LoggingDispatcher` at all. I was thinking something along the line of "If you would like to customize logging further, see \[creating and using your own dispatcher\]"

Comment (martinzlocha) [reply docs/content/ecs/logging.md]: Given that it exists I think we should mention it. If we don't want it at all then we should remove it from here and the docs.

File: docs/content/ecs/logging.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jessicafalk) [reply docs/content/ecs/logging.md]: that's the official name. Take it up with @ElleEss ;)
